preferences/windows/2k12/kustomization.yaml

```diff
@@ -8,7 +8,5 @@ resources:
 components:
   - ./metadata
   - ./requirements
-  - ../../components/deprecated
-  - ../../components/secureboot
```

Comment:
No more secureboot?

Reply:
No, @ksimon1 and I spend some effort trying to make it work, without success.